content/01-getting-started/02-setup-prisma/01-add-to-existing-project.mdx

```diff
@@ -451,7 +451,7 @@ To get started with Prisma Client, you need to install it as a node module:
 npm install @prisma/client
 ```
 
-Notice that the [`@prisma/client` node module](../../reference/tools-and-interfaces/prisma-client/configuring-the-prisma-client-api#the-prisma-client-npm-module) is a bit different from "conventional" node modules (e.g. a library like [`lodash`](https://lodash.com/)). When installing the module with `npm install @prisma/client`, npm only downloads an empty package into the `node_modules/@prisma/client` directory. The package is then "filled" with the actual library with the following command of the Prisma CLI:
+Notice that the [`@prisma/client` node module](../../reference/tools-and-interfaces/prisma-client/configuring-the-prisma-client-api#the-prisma-client-npm-module) references a folder named `.prisma\client`. The `.prisma\client` folder contains your generated Prisma client, and is modified each time you change the schema and run the following command:
```

Comment:
The target of this link does not really exist on that page.

(Noticed because the wording "`@prisma/client` node module" sounded strange to me. Npm has packages usually.)

(Also applies to all the other files)